### DISCUSSION TITLE
Truffle Dashboard Connecting Wallet Issue

### DISCUSSION THREAD

**Discussion by CoachSteve** [ConsenSys-archive/truffle#5111] [General Q&A]
I have had this issue twice, but unfortunately I do not know what causes it to come and go.  I am using truffle dashboard and metamask.  Although yesterday I was able to select the polygon mainnet network fine, today the dashboard is automatically reverting to ethereum mainnet and showing a different wallet address than mine.  This is the second time it as happened.  The first time I was at a loss so I just took a break from my project.  After a few weeks, it seemed good, then it happened again.  Any ideas?

**Comment by gnidan** [comment]
Hm, really weird. Is it possible that you have multiple wallets running, like MetaMask and Frame? Truffle Dashboard should just use whatever in-page provider it is that MetaMask gives to every dapp.

**Reply by CoachSteve** [reply]
Hello again.  Although my collection is just coming out, the purpose of the project is to show how much information is out there for us to take advantage of - if we take the time to find it.  I am dropping nft's to those who have helped in some way get this collection finished and although it may seem you did little, your input helped tremendously!  So, if you share an Ethereum wallet address, I would love to drop you an official The Musketeersnft original.

**Comment by CoachSteve** [comment]
First, thank you so much for responding.  I am not sure what Frame is even.  I have metamask, coinbase wallet and phantom.  The odd part is the way it works then doesn't work.  I haven't tried it tonight again yet.  Even if it does work, I have zero confidence that it will work again afterwards.

**Comment by CoachSteve** [comment]
THANK YOU!!!!!!  Although I do not have Frame, when you mentioned other wallets, I figured I would take a look.  I have/had Metamask, Phantom, Binance and Coinbase and after checking each one separately, the culprit is the Coinbase Wallet!  After switching that extension off, truffle dashboard connects to my metamask perfectly.  I have no idea why/how it was jumping back and forth, but I really do appreciate your input and help.  When my nft's get famous, I'll come back and get your wallet so I can drop one to you! :)
